ang/civicase/case/details/directives/case-details.directive.js

```diff
@@ -16,8 +16,27 @@
 
   module.controller('civicaseCaseDetailsController', civicaseCaseDetailsController);
 
+  /**
```

Comment:
@deb1990 tested this, ESLint complains that the file is missing a few parameters first. If you do nothing and commit the file it will automatically add the missing parameter names, but it won't add the type or the description and it will commit as normal without issues.

If you check the file again you'll see ESLint complaining about the lack of type and description, but it lets you commit the file.

I think we need to do the following:

1. If there are lint issues, they should not be automatically fixed when committing the file.
2. If there are lint issues it should not commit at all.

Reply:
@reneolivo Thats a good idea. To fix this, please change this line https://github.com/compucorp/uk.co.compucorp.civicase/blob/de0f6ac64a54df17d61c7ab70e534587df297bb4/.lintstagedrc#L3
we just need to remove the `--fix` part.